charters/PM/charter.md

```diff
@@ -0,0 +1,60 @@
+# PM Team Charter
+
+## Description
+
+The Data Coordination Platform (DCP) PM team is responsible for the oversight of the DCP *as product*, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
+
+## Definitions
+
+* A **Thematic Roadmap** identifies and expresses customer needs as a small collection of themes or strategic objectives which:
+    * Focuses the DCP community on a documented set of high-level customer priorities
+    * Excites users about the future of DCP
+
+* **Request for Comments (RFC)** is the transparent process that manages all significant enhancements to the DCP software technical and governance documentation, both normative and informative
+
+## In-scope
+
+*Governance* 
+
+* Develop standard processes and best practices for DCP community governance including (but not limited to):
+    * Program and technical oversight, decision making, and conflict resolution
+    * Charters for DCP projects
+    * Enhancements (RFC) to DCP software or governance 
+    * Planning
+    * Communication
+
+* Review and approve non-technical charters for DCP projects
+
+* Act as final arbiter for escalated conflicts from the DCP community
+
+*Communication*
+* Coordinate regular outreach and announcements to the DCP community and the HCA Science Leadership
+
+* Primary liaison between the HCA Science Leadership and the DCP community facilitated by the *Science Program Manager*
+
+*Planning*
+
+Strategic 
+
+* Identify and maintain a repository of user requirements, use cases, and journeys informed by *DCP UX* research
+
+* Facilitate a `<insert cadence>` planning process to solicit input from the HCA Science and DCP communities
+    * Publish a Thematic Roadmap RFC based on that input that also adddresses strategic technical objectives from the DCP Architecture team
```

Comment:
Can you define "thematic roadmap" here?

Reply:
@briandoconnor - it's included in the Definitions section:

* A **Thematic Roadmap** identifies and expresses customer needs as a small collection of themes or strategic objectives which:
    * Focuses the DCP community on a documented set of high-level customer priorities
    * Excites users about the future of DCP